### PR TITLE
Updated apiary docs. Edits on Drop-in Gift Cards

### PR DESCRIPTION
**Pull request by Tim-Jordison**
- Updated apiary docs to link to drop-in gift card docs.
- Edits on drop-in gift cards.

### CODE REVIEW

File: apiary-sources/sections/introduction-master.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ conceptual model, common use-cases, and common API calls with examples.
 
 Contact us anytime at hello@lightrail.com —we are here to help you solidify your use-cases and implementation.
 
+## Looking for Lightrail's Drop-In Gift Card Solution?
+Lightrail's Drop-in Gift Card solution that makes it very easy to add gift cards and account credits to your application. The solution is component based, using snippets that can be included for gift card sending and redemption. It requires minimal updates to your existing checkout process, which is easy to do using our [client libraries](https://github.com/Giftbit/Lightrail-API-Docs/blob/master/docs/client-libraries.md).
```
Comment (tjukes) [inline apiary-sources/sections/introduction-master.md]: "Lightrail's Drop-in Gift Card solution ~that~ makes it..."

also maybe:
"...using widgets to handle gift card sending and redemption."

File: apiary.apib [hunk 2]
```diff
@@ -10,12 +10,10 @@ conceptual model, common use-cases, and common API calls with examples.
 
 Contact us anytime at hello@lightrail.com —we are here to help you solidify your use-cases and implementation.
 
-## Looking for Lightrail's Drop In Gift Card Solution?
-Lightrail is developing a drop in gift card solution that makes it very easy to add gift cards and account credits to your application.
-The solution is component based, using snippets that can be included for gift card sending and redemption.
-Checkout requires minimal updates to your existing checkout process and is easy to do using our existing libraries.
+## Looking for Lightrail's Drop-In Gift Card Solution?
+Lightrail's Drop-in Gift Card solution that makes it very easy to add gift cards and account credits to your application. The solution is component based, using snippets that can be included for gift card sending and redemption. It requires minimal updates to your existing checkout process, which is easy to do using our [client libraries](https://github.com/Giftbit/Lightrail-API-Docs/blob/master/docs/client-libraries.md).
```
Comment (tjukes) [inline apiary.apib]: See prev comment -- not sure which file needs to be updated?

File: docs/quickstart/drop-in-gift-cards.md [hunk 3]
```diff
@@ -1,24 +1,29 @@
...
 
+If at any point you want to see a working example of the entire Drop-in Gift Card solution, check out our [sample app](https://github.com/Giftbit/stripe-integration-sample-webapp). 
+
 ## Step 1: Selling Gift Cards
 The Gift Card Purchase Widget allows your customers to purchase gift cards from your site. 
 Lightrail powers the entire gift card purchase and delivery flow. 
 ![Gift card purchase widget](assets/purchase-widget.png)
 
-Add the following snippet to your site: 
+What you see here is our fictional brand called Rocktship. Once set up with our Drop-in solution, you will see your branding instead.
```
Comment (tjukes) [inline docs/quickstart/drop-in-gift-cards.md]: "Rocktship" -> "Rocketship"

File: apiary.apib (hunk 2 above)
Comment (Tim-Jordison) [reply apiary.apib]: Good catch, yep. It's the previous file: `introduction-master.md`. 
`apiary.apib` is the compiled file.

**Pull request merged**